cmd/trtl/main.go

```diff
@@ -530,6 +546,195 @@ func initPeersClient(c *cli.Context) (err error) {
 // Trtl (DB) Client Functions
 //===========================================================================
 
+// migrateCerts creates certificate records in the trtl database based on the existing certreqs.
+func migrateCerts(c *cli.Context) (err error) {
+	ctx, cancel := profile.Context()
+	defer cancel()
+
+	// Open the report file if requested
+	var report *os.File
+	if c.String("report") != "" {
+		if report, err = os.Create(c.String("report")); err != nil {
+			return cli.Exit(fmt.Errorf("could not open report file %s: %s", c.String("report"), err), 1)
+		}
+		defer report.Close()
+	}
+
+	// Get the list of certreqs
+	req := &pb.CursorRequest{
+		Namespace: wire.NamespaceVASPs,
+	}
+	var stream pb.Trtl_CursorClient
+	if stream, err = dbClient.Cursor(ctx, req); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	var migrated, skipped int
+	var migrateErr error
```

Comment:
I think it might be better to use a `multierror` here - it feels like there are a couple of places where we might want to see multiple nested errors. It will also help protect us from the break logic.